frontend/index.html

```diff
@@ -0,0 +1,63 @@
+            <button class="top-bar-button"> NÁVOD K POUŽITÍ </button>
+        </div>
+        <img class="man-logo" src="man_logo_batch.svg" alt="man-logo">
+    </div>
+    <div class="center-buttons">
+        <div class="mechanic-buttons">
+            <button class="middle-button" id="mechanic_0">MECHANIC 1</button>
+            <button class="middle-button" id="mechanic_1">MECHANIC 2</button>
+            <button class="middle-button" id="mechanic_2">MECHANIC 3</button>
+            <button class="middle-button" id="mechanic_3">MECHANIC 4</button>
+            <button class="middle-button" id="mechanic_4">MECHANIC 5</button>
+            <button class="middle-button" id="mechanic_5">MECHANIC 6</button>
+            <button class="middle-button" id="mechanic_6">MECHANIC 7</button>
+            <button class="middle-button" id="mechanic_7">MECHANIC 8</button>
+            <button class="middle-button" id="mechanic_8">MECHANIC 9</button>
+            <button class="middle-button" id="mechanic_9">MECHANIC 10</button>
+            <button class="middle-button" id="mechanic_10">MECHANIC 11</button>
+            <button class="middle-button" id="mechanic_11">MECHANIC 12</button>
+            <button class="middle-button" id="mechanic_12">MECHANIC 13</button>
+            <button class="middle-button" id="mechanic_13">MECHANIC 14</button>
+            <button class="middle-button" id="mechanic_14">MECHANIC 15</button>
+            <button class="middle-button" id="mechanic_15">MECHANIC 16</button>
+            <button class="middle-button" id="mechanic_16">MECHANIC 17</button>
+            <button class="middle-button" id="mechanic_17">MECHANIC 18</button>
+            <button class="middle-button" id="mechanic_18">MECHANIC 19</button>
+            <button class="middle-button" id="mechanic_19">MECHANIC 20</button>
+            <button class="middle-button" id="mechanic_20">MECHANIC 21</button>
+            <button class="middle-button" id="mechanic_21">MECHANIC 22</button>
+            <button class="middle-button" id="mechanic_22">MECHANIC 23</button>
+            <button class="middle-button" id="mechanic_23">MECHANIC 24</button>
+        </div>
+        <div class="other-buttons">
+            <button class="middle-button" id="technique_0">TECHNIQUE 1</button>
+            <button class="middle-button" id="technique_1">TECHNIQUE 2</button>
+            <button class="middle-button" id="technique_2">TECHNIQUE 3</button>
+            <button class="middle-button" id="technique_3">TECHNIQUE 4</button>
+            <button class="middle-button" id="technique_4">TECHNIQUE 5</button>
+            <button class="middle-button" id="other">OSTATNÍ...</button>
+        </div>
+    </div>
+    <div class="bottom-bar">
+        <button class="file-chooser">VÝBĚR SOUBORU</button>
+        <input type="image" class="truck" src="send_truck.svg" alt="truck-icon">
+    </div>
+</body>
```

Comment:
Nemáš uzavřenej <html> tag, chybí newline na konce souboru (AFAIK VS Code má na to nějaký nastavení, aby ho tam přidával autmaticky